addons/delivery/models/delivery_carrier.py

```diff
@@ -187,3 +189,16 @@ def fixed_get_tracking_link(self, picking):
 
     def fixed_cancel_shipment(self, pickings):
         raise NotImplementedError()
+
+    def merge_pdf(self, label_name, label_data):
```

Comment:
Can you make this method a bit generic and move it to `odoo/tools/pdf.py` (to be created)?
Actually, it has no link with `delivery.carrier` objects, it is *used by* `delivery.carrier`s